package.json

```diff
@@ -3,21 +3,19 @@
   "version": "1.0.1",
   "description": "generates a cryptographically strong random uid",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "files": [
     "lib"
   ],
   "scripts": {
-    "test": "xo && ava"
+    "build": "tsc",
+    "test": "ava"
```

Comment:
Probably test should be `yarn build & ava`

Reply:
I don't think Travis CI has yarn installed.

If `test` becomes both build and test steps then I'll need to revert the [travis.yml](https://github.com/zeit/uid-promise/pull/26/files#diff-354f30a63fb0907d4ad57269548329e3) changes or else it will build twice.

Reply:
I fixed this 🔧